pkg/config/unexpectedunicodefinder_test.go

```diff
@@ -0,0 +1,72 @@
+package config
+
+import "testing"
+
+func TestCleanStrings(t *testing.T) {
+	cleanString := "container_collect_some"
+
+	res := FreeFromUnexpectedUnicode([]byte(cleanString))
+	if len(res) != 0 {
+		t.Errorf("Expected no unexpected codepoints, but found some: %v", res)
+	}
+}
+
+func TestDirtyStrings(t *testing.T) {
+	dirtyString := "‪container_collect_some"
```

Comment:
Would it make sense to use unicode escapes instead of embedding it literally? It would be more clear what is being tested exactly, as well as help preserve the test text if the file needs to be edited.

Reply:
Yes, good suggestion. Making this change.